docs/networks/node-ops/access-onchain-data/access-nodes/access-node-configuration-options.md

```diff
@@ -0,0 +1,241 @@
+
+### Download the root checkpoint
+
+You will need to download checkpoint files for the corresponding height which is the same as was for downloading the root protocol state snapshot. This is important cause in other cases you will get the error described in the Troubleshooting section.
+
+Once the files are downloaded, you can either move them to `/bootstrap/execution-state/` within the node’s bootstrap directory or put them in any mounted directory and reference the location with this cli flag: `--execution-state-checkpoint=/path/to/root.checkpoint`. The naming of files should be `root.checkpoint.*`.
+
+# Configure the node to run the indexer
+
+Now you have the execution sync setup and the root checkpoint in place, it’s time to configure the node to index all of the data so it can be used for script execution.
+
+There are 2 cli flags that you will need to add:
+
+- `--execution-data-indexing-enabled=true` This will enable the indexer.
+- `--execution-state-dir` This defines the path where the registers db will be stored. A good default is on the same drive as the protocol db. e.g. `/data/execution-state`
+
+# Start your node
+
+Now that all of the settings to enable indexing are in place, you can start your node.
+
+At a minimum, you will need the following flags:
+
+```
+--execution-data-indexing-enabled=true
+--execution-state-dir=/data/execution-state
+--execution-data-sync-enabled=true
+--execution-data-dir=/data/execution-data
+```
+
+For better visibility of the process, you can also add
+
+`-p 8080:8080` - export port 8080 from your docker container, so you could inspect the metrics
+
+`--loglevel=info` - for checking logs.
+
+Notes on what to expect:
+
+- On startup, the node will load the checkpoint into the `execution-state` db. For `devnet48`, this takes 20-30 min depending on the node’s specs. For `mainnet24`, it takes >45 min. The loading time will increase over time. You can follow along with the process by grepping your logs for `register_bootstrap`.
+- After the checkpoint is loaded, the indexer will begin ingesting the downloaded execution data. This will take several hours to days depending on if the data was already downloaded and the hardware specs of the node.
+- If your node already had all the data, it will index all of it as quickly as possible. This will likely cause the node to run with a high CPU.
+
+When you restart the node for the first time with syncing enabled, it will sync execution data for all blocks from the network.
+
+# Troubleshooting
+
+- If the root checkpoint file is missing or invalid, the node will crash. It must be taken from the same block as the `root-protocol-state-snapshot.json` used to start your node.
+- If you don’t set one the `--execution-data-dir` and `--execution-state-dir` flags, the data will be written to the home directory inside the container (likely `/root`). This may cause your container to run out of disk space and crash, or lose all data each time the container is restarted.
```

Comment:
```suggestion
- If you don’t set the `--execution-data-dir` and `--execution-state-dir` flags, the data will be written to the home directory inside the container (likely `/root`). This may cause your container to run out of disk space and crash, or lose all data each time the container is restarted.
```